spiffs_file_preprocessor/txt_processor.c

```diff
@@ -50,4 +58,10 @@ void process_txt(const char *infile, const char *outdir)
     FILE* outFile = fopen(outFilePath, "wb");
     fwrite(txtInStr, newSz, 1, outFile);
     fclose(outFile);
+
+    char rmv = 't';
+    char tst[] = "test string";
```

Comment:
I don't think you meant to leave this in, but I like that you used it to test

Reply:
oops! you are correct, removing now.